index.html

```diff
@@ -14,19 +14,36 @@
 </head>
 
 <body>
+
     <header>
         <a href="#">Kemboi</a>
         <nav>
             <img class="header-icon" src="images/union.svg" />
             <ul>
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
                 <img class="header-mail" src="images/mail.svg" alt="Email" />
             </ul>
         </nav>
     </header>
 
+    <div class="navMenu" id="navMenu">
+        <button class="menuBtn">
+            <img src="./images/cross.svg" alt="">
+        </button>
+        <ul>
+            <li>
+                <a href="#portfolio">Portfolio</a>
+                <hr>
+            </li>
+            <li><a href="#about">About</a></li>
+            <hr>
+            <li><a href="#contact">Contact</a></li>
+            <hr>
+        </ul>
```

Comment:
👍 Great work on implementing the navigation functionality! The navigation is working correctly, and when the user clicks on any nav item, the correct section of the page is displayed. However, there are a couple of improvements needed:

- [ ] As per the project requirement, when the user clicks on any option of the mobile menu, the mobile menu should disappear or close. Currently, the mobile menu does not close upon user selection. Please update your code to hide the mobile menu when a menu option is clicked, similar to how you implemented it for the close icon.

- [ ] Additionally, to match the design requirements, please ensure that the navigation has proper styling, including font size, font weight, line height, spacing, color, border and layout. This will give your navigation a polished and consistent look, aligning it with the design.
![image](https://github.com/okemboi/portfolio/assets/55840999/44bd3b24-f7bb-440d-ada7-86b84b5eb589)

By addressing these points, your navigation will provide a more seamless and user-friendly experience. Keep up the good work, and happy coding! 🚀💻🌟